php/README.md

```diff
@@ -96,7 +96,11 @@ We recommend that you embed the source code in your container and copy it to the
 |  `nginx.image.repository` | The image repository to pull from | `nginx` |
 |  `nginx.image.tag` | The image tag to pull | `1.15.1-alpine` |
 |  `nginx.image.pullPolicy` | Image pull policy | `IfNotPresent` |
+|  `nginx.host` | Default host of NGINX | `localhost` |
```

Comment:
@cw-niinuma 
Where are you using this value?

Reply:
@cw-ozaki Don't use it. I'll Delete it.